packages/exo/src/get-interface.js

```diff
@@ -18,7 +18,7 @@ export const GET_INTERFACE_GUARD = '__getInterfaceGuard__';
 /**
  * @template {Record<PropertyKey, CallableFunction>} M
  * @typedef {{
- *   [GET_INTERFACE_GUARD]: () =>
+ *   [GET_INTERFACE_GUARD]?: () =>
```

Comment:
attn @erights

Reply:
As discussed, I think this is good enough. Resolving.